cli/src/diff_util.rs

```diff
@@ -73,15 +75,18 @@ pub struct DiffFormatArgs {
     /// Generate diff by external command
     #[arg(long)]
     pub tool: Option<String>,
+    /// Number of lines of context to show.
```

Comment:
nit: omit period in doc string of `clap::Args`

Reply:
Done